content/general/limits.textile

```diff
@@ -0,0 +1,314 @@
+
+h3(#token-size). Token request size
+
+The token request size is the maximum size of a signed token request that will be accepted by the Ably platform.
+
+h2(#connection). Connection limits
+
+Connection limits relate to the "realtime connections":/realtime/connection to Ably from your account.
+
+|_=. Limit |_=. Free |_=. Self-service |_=. Business |_=. Enterprise |
+| "Connection rate":#connection-rate (per second) | 20 | >=50 | >=50 | Custom |
+| "Number of channels":#connection-channels (per connection) | 50 | 200 | 200 | Custom |
+| "Outbound message rate":#outbound-message (per second) | 15 | 50 | 50 | Custom |
+| "Inbound message rate":#inbound-message (per second) | 15 | 50 | 50 | Custom |
+| "Connection state recovery":#connection-state |\4=. 2 minutes |
+
+h3(#connection-rate). Connection rate
+
+The connection rate is the number of new "realtime connections":/realtime/connection that can be made to Ably per second.
+
+The limit is calculated based on the hard limit for "peak connections":#peak-connections, with a minimum value of 50 per second for self-service and business accounts.
+
+If the connection rate is exceeded, a "global per second limit":#global-time will be applied to new connection attempts.
+
+h3(#connection-channels). Number of channels
+
+The number of channels per connection are the number of channels each client can be attached to on a "realtime connection":/realtime/connection.
+
+The number of channels per connection is a "quantity-based limit":#quantity-rates. If a connection attempts to exceed the limit on the number of channels it is attached to, the attachment will fail and the error code @90010@ will be returned.
+
+h3(#outbound-message). Outbound message rate
+
+The outbound message rate is the maximum number of messages that can be published per second on a "realtime connection":/realtime/connection.
+
+If the outbound message rate is exceeded for a connection, a "local per second limit":#time will be applied to that connection.
+
+h3(#inbound-message). Inbound message rate
+
+The inbound message rate is the maximum number of messages that can be received per second on a "realtime connection":/realtime/connection.
+
+If the inbound message rate is exceeded for a connection, a "local per second limit":#time will be applied to that connection.
+
+h3(#connection-state). Connection state recovery
+
+Connection state recovery is the duration the Ably service will keep a connection alive for if the client connection drops. This accounts for scenarios such as where a client is switching between WiFi and 4G, or entering a tunnel. If the client reestablishes its connection within the allowed duration, all messages and presence events will be replayed to that client in order.
```

Comment:
"Connection state recovery is the duration":  connection state recovery is a feature, or is functionality. Connection State TTL (time to live) is the duration. I think this section should be renamed Connection State TTL.

Reply:
Renamed the section, thanks.